src/unicode.rs

```diff
@@ -700,6 +701,57 @@ impl CharExt for char {
         (0x0FE00..=0x0FE0F).contains(&ch) || // VARIATION SELECTOR - 1..16
         (0xE0100..=0xE01EF).contains(&ch)    // VARIATION SELECTOR - 17..256
     }
+
+    fn vertical(self) -> Option<char> {
+        Some(match u32::from(self) >> 8 {
+            0x20 => match self {
+                '\u{2013}' => '\u{fe32}', // EN DASH
```

Comment:
Would you mind keeping hex coded? Unicode notation is too ugly/verbose.

Reply:
Sure.

Reply:
Only on the left side of the match though, right?

Reply:
Ufff... you're returning char. Ok, leave it as is.

Reply:
Well, the method is only used with `u32` at the call site. But I thought conceptually it would fit well into `CharExt`.

Reply:
Yes, we should prefer char to u32. It's fine.